app.py

```diff
@@ -1,37 +1,42 @@
-from flask import Flask, request, jsonify
+# from flask_sqlalchemy   import SQLAlchemy
+# from flask_marshmallow import Marshmallow
+# import code
+# code.interact(local=dict(globals(), **locals()))
+import json
+import os
+from flask import Flask, jsonify, request
 import requests
 from dotenv import load_dotenv
-import os
 load_dotenv()
 
+
 # Init app
 app = Flask(__name__)
 
-# Get NREL solar system data
-@app.route('/results/<fe_json>', methods=['GET'])
-def get_solar_system(fe_json):
-    results = NREL(fe_json)
-    return results.get()
 
+# Get solar system data from NREL API
+@app.route('/results', methods=['POST'])
+def get_solar_system():
+    results = NREL(request.json)
```

Comment:
Awesome job getting this to work!

Reply:
Thanks man!  I appreciate the help in getting to this point!